### DISCUSSION TITLE
is it required to use Finrl-meta instead of finrl

### DISCUSSION THREAD

**Discussion by krishdotn1** [AI4Finance-Foundation/FinRL-Meta#322] [Q&A]
I have  seen agents and some dataprocessors and some enviroments is exactly same. so I'm confused what was the purpose of creating diffrent repo for same. 

I can understand in finrl-meta there is more dataprocessor and enviroment but what about agents? finrl and finrl-meta have same agents. can we simply import those agents from finrl ? @zhumingpassional @ZiyiXia please reply :)

**Comment by zhumingpassional** [comment]
Hi.

finrl-meta is the financial datasets of finrl. We will move the data sources of finrl-meta if they are stable. 

the agents are almost the same, both of which use SB3 and elegantrl.
